PiwikTracker/PiwikTracker.swift

```diff
@@ -36,6 +36,8 @@ final public class PiwikTracker: NSObject {
 
     internal var dimensions: [CustomDimension] = []
     
+    @objc public var useDefaultCustomVariables: Bool = false
+    private lazy var cvars: [CustomVariable] = getDefaultCVars()
```

Comment:
I don't like the `cvars` variable name here to much. We usually don't use abbreviations. What do you think about using `customVariables` here? This then shows another minor issue. It is imho hard to distinguish (just by the variable names) between the `cvars` and the `customVariables` that you implemented as a slice from these here.

What do you think about removing the `useDefaultCustomVariables` completely and let the users of the SDK add the "default" variables themselves? Exposing a `defaultVariables` property on the `CustomVariable` for example could help here. This way, the code here in the SDK is much cleaner and it is more direct for the user.